projects/aca-folder-rules/src/lib/manage-rules/manage-rules.smart-component.ts

```diff
@@ -103,10 +103,13 @@ export class ManageRulesSmartComponent implements OnInit, OnDestroy {
     this.selectedRule = rule;
   }
 
-  openNewRuleDialog() {
+  openNewRuleDialog(model = {}) {
```

Comment:
Maybe rename this method since it's used for both the create and edit dialogs now

Reply:
I thought it is already universal, as naming is not strictly connected to rule creation or update. How about just `openRuleDialog()`?

Reply:
I was thinking something that covers rule updates as well as creation. `openCreateUpdateRuleDialog()` could work